index.html

```diff
@@ -6,26 +6,41 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/main.css" />
     <link rel="icon" href="./favicon.ico" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
-          <li>
-            <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
-            ></a>
-          </li>
-        </ul>
-      </nav>
-    </header>
+      <div class="header">
+         <h1 id="logo">Aziz Mejri</h1>
+         <div>
+           <button type="button" class="menu-bars">
+             <i class="fas fa-bars"></i>
+           </button>
+         </div>
+
+         <nav id="desktop-nav">
+           <ul>
+             <li><a class="nav-link" href="#projects">Portoflio</a></li>
+             <li><a class="nav-link" href="#about-me">About</a></li>
+             <li>
+               <a class="nav-link" href="#contact">Contact <i class="far fa-envelope"></i></a>
+             </li>
+           </ul>
+         </nav>
+       </div>
+
+       <nav id="mobile" class="menu">
+         <div></div>
+         <ul class="list-menu">
+           <li><a href="#" rel="noopener noreferrer">Hello</a></li>
+           <li><a href="#" rel="noopener noreferrer">Portfolio</a></li>
+           <li><a href="#" rel="noopener noreferrer">About</a></li>
+           <li><a href="#" rel="noopener noreferrer">Contact</a></li>

```

Comment:
I noticed that after clicking on any menu option, It doesn't refer that to a particular part of the page, Kindly fix it.
"When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed."